client/checkout/api/index.js

```diff
@@ -344,12 +344,12 @@ export default class WCPayAPI {
 		} )
 			.then( ( response ) => {
 				if ( 'failure' === response.result ) {
-					throw response.messages;
+					throw { message: response.messages };
```

Comment:
If there is a specific object type to use here, happy to change it. I couldn't find any other example wrapping up the message.

Reply:
```suggestion
					throw new Error( response.messages );
```

I think since `response.messages` will be a string in this situation, you could throw a JS Error instead and `showError` would be able to unpack the string from the error's `message` property. Works for me when I tested it and might look a little cleaner, but I leave it in your hands.

Reply:
Done!